clients/ts/signalr/src/Errors.ts

```diff
@@ -43,3 +45,23 @@ export class TimeoutError extends Error {
         this.__proto__ = trueProto;
     }
 }
+
+/** Error thrown when an action is aborted. */
+export class AbortError extends Error {
```

Comment:
This wasn't part of the original change, looks like something from 2.2 that's leaking over.